TAKING ME
[{u'calltype': u'amd64', u'realsz': 333, u'diff': u'NEW', u'name': u'sym.enqueue', u'cc': 3, u'indegree': 0, u'nargs': 0, u'difftype': u'new', u'edges': 10, u'outdegree': 10, u'nlocals': 14, u'cost': 115, u'datarefs': [24, 6340888, 6340864, 4233256], u'callrefs': [{u'type': u'C', u'addr': 4225568, u'at': 4223353}, {u'type': u'C', u'addr': 4225568, u'at': 4223394}, {u'type': u'C', u'addr': 4225568, u'at': 4223435}, {u'type': u'C', u'addr': 4222400, u'at': 4223456}, {u'type': u'C', u'addr': 4223600, u'at': 4223595}, {u'type': u'C', u'addr': 4222592, u'at': 4223615}, {u'type': u'C', u'addr': 4223632, u'at': 4223574}, {u'type': u'C', u'addr': 4223760, u'at': 4223579}, {u'type': u'J', u'addr': 4223544, u'at': 4223533}, {u'type': u'C', u'addr': 4223664, u'at': 4223539}, {u'type': u'C', u'addr': 4223696, u'at': 4223563}, {u'type': u'J', u'addr': 4223579, u'at': 4223568}], u'offset': 4223296, u'ebbs': 2, u'nbbs': 9, u'type': u'sym', u'size': 334}]
ERR. Target function sym.enqueue0 does not match sym.enqueue
Failed to patch all expected patches: 45  total patched: 90
